bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -57,7 +60,8 @@ def _string_from_json(value, _):
 def _bytes_from_json(value, field):
     """Base64-decode value"""
     if _not_null(value, field):
-        return base64.decodestring(value)
+        return base64.decodestring(
+            _to_bytes(value) if isinstance(value, six.text_type) else value)
```

Comment:
@tseaver The `if isinstance()` check happens within `_to_bytes`, i.e. that's what it's for

Sorry I wasn't clear on that.